package.json

```diff
@@ -35,12 +35,11 @@
     "grunt": "1.0.2",
     "grunt-compare-size": "0.4.2",
     "grunt-contrib-connect": "1.0.2",
-    "grunt-contrib-jshint": "1.1.0",
     "grunt-contrib-nodeunit": "2.0.0",
     "grunt-contrib-qunit": "2.0.0",
     "grunt-contrib-uglify": "2.3.0",
     "grunt-contrib-watch": "1.1.0",
-    "grunt-jscs": "3.0.1",
+    "grunt-eslint": "^20.2.0",
```

Comment:
Just a small comment, can we use fixed versions here?

Reply:
Sorry, hadn't noticed you're using fixed versions for dependencies. Fixed now.